source/concurrent/Thread.ooc

```diff
@@ -64,9 +64,9 @@ version (unix || apple) {
 }
 
 version (windows) {
-	include windows
+	include inttypes
 
-	ThreadId: cover from DWORD {
+	ThreadId: cover from UInt {
```

Comment:
You should probably change `DWORD: extern cover from Long` in `system/os/win32.ooc` instead.